src/features/ajax/aggregate/gql.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @typedef {object} GQLMetadata
+ * @property {string} operationName Name of the operation
+ * @property {string} operationType Type of the operation
+ * @property {string} operationFramework Framework responsible for the operation
+ */
+
+/**
+ * @param {string|Object} gql The GraphQL object body sent to a GQL server
+ * @returns {GQLMetadata}
+ */
+export function parseGQL (gql) {
```

Comment:
Just a general though here but it would be nice if this module had a single export that took in the necessary parameters and returned the metadata. The rest of the function should not be exported and can be tested through the public interface.